api/processors/group.py

```diff
@@ -29,6 +29,13 @@ def start(cls, group: Group) -> None:
         cls._create_log_events_start(group)
         cls._set_statuses_start(group)
 
+    @classmethod
+    @transaction.atomic
+    def abort(cls, group: Group) -> None:
+        cls._create_log_events_abort(group)
+        cls._mark_group_linked_objects_as_former(group)
```

Comment:
```suggestion
        cls._move_related_people_to_former(group)
```
Just a naming nitpick here. We're not actually marking anyone, we're literally moving them from one set to another.